### PR TITLE
fix(leafletData unresolveMap): argument name renamed scopeId to mapId

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

File: src/services/leafletData.js [hunk 1]
```diff
@@ -27,8 +27,8 @@ angular.module('ui-leaflet').service('leafletData', function (leafletLogger, $q,
         _private[itemName] = {};
     });
 
-    this.unresolveMap = function (scopeId) {
-        var id = leafletHelpers.obtainEffectiveMapId(_private.map, scopeId);
+    this.unresolveMap = function (mapId) {
+        var id = leafletHelpers.obtainEffectiveMapId(_private.map, mapId);
```
Comment (nmccready) [inline src/services/leafletData.js]: Main fix is here.

**Comment by nmccready** [comment]
I am trying to remove the bithound integration without much success. I have contacted bithound for help. 

Anyway I will get to the bottom of what is going on with Travis.

**Comment by nmccready** [comment]
@elesdoar merging once Travis passes

**Comment by elesdoar** [comment]
OK

**Pull request merged**